core/internal/cltest/cltest.go

```diff
@@ -1465,3 +1486,20 @@ func MustBytesToConfigDigest(t *testing.T, b []byte) ocrtypes.ConfigDigest {
 	}
 	return configDigest
 }
+
+func MockSubscribeToLogs(ethClient *mocks.Client, sub *mocks.Subscription) chan<- models.Log {
+	logs := make(chan<- models.Log, 1)
+	ethClient.On("SubscribeFilterLogs", mock.Anything, mock.Anything, mock.Anything).
+		Return(sub, nil).
+		Run(func(args mock.Arguments) { // context.Context, ethereum.FilterQuery, chan<- types.Log
+			logs = args.Get(2).(chan<- types.Log)
```

Comment:
we used a channel of channels in other places to feed logs in, it seems simpler to just use a single channel like this